README.md

```diff
@@ -22,20 +24,24 @@ pip install -r requirements.txt
 
 ## Running tests
 
-Execute the following command from the root directory to run the tests.
+Execute the following command from the project root directory to run the tests.
 
 ```
 python -m unittest discover
 ```
 
-## Running LegacyMan parser program
+## Running LegacyMan parser program (against mock data)
 
-Execute the following command from the root directory to run program.
+Execute the following command from the project root directory to run program.
 
 ```
-python -m legacyman_parser.hello
+python -m legacyman_parser.parser data/PlatformData/PD_1.html
 ```
 
-### Generate discrepancy reports
+This will
+* generate the json file to be imported into the Data Browser application
+* generate a discrepancy report of inaccessible urls
```

Comment:
let's limit it to "generate a discrepancy report"

Reply:
Okay

Reply:
Done.